README.md

```diff
@@ -39,9 +51,13 @@ To use RA-TLS wrapper, you must have Go installed on your system. You can then i
 go get github.com/konvera/gramine-ratls-golang
 ```
 
+## Initialisation
```

Comment:
Missing reference to `LoadRATLSVerifyLibs`.
I think it makes sense to load all libs in one method, if the libs for verifying are present then those for creating the cert are present as well.